src/content/protocols/aave.md

```diff
@@ -0,0 +1,885 @@
+| GHO                                                               | mint                                   | A facilitator can mint GHO. Facilitators are approved by the `FACILITATOR_MANAGER_ROLE` owner. Currently GHO can be created in 4 ways. In a flash loan facilitated by `GHOFlashMinter`, GHO tokens can be borrowed (collateralised borrow position), the GHO Stability Modules which allow to trade 1 GHO against 1 USDC or USDT at a fixed rate and the `GhoDirectMinter` which allows the governance to mint and supply GHO in to the Aave Market. The privilege to mint GHO must be sufficiently protected to prevent loss of funds via dilution. The governance has control over all listed facilitators via this contract `Executor_lvl1`.                                                                                                                                                                                                                                                                                                                                                                                                                                            | GhoAToken, GHOFlashMinter, GSMUSDC, GSMUSDT, GhoDirectMinter                                                                                                                 |
+| GHO                                                               | burn                                   | A facilitator can beside minting also burn GHO tokens. See for more context the row above.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
```

Comment:
does this mean that cross chain messaging could effectively be DOS-ed by continuously calling `retryEnvelope` to effectively reset the counter to 0 over and over again?

Reply:
seems not, good then